### DISCUSSION TITLE
Stretch property doesn't work sometimes?

### DISCUSSION THREAD

**Discussion by MattTheCuber** [revolist/revogrid#622] [General]
**Describe the issue**
Using the `stretch="true"` property doesn't seem to have any affect in most cases. I am running Vue 3.5.8 and datagrid 4.10.12 and it is doing nothing on my machine.

When I go to any examples on code sandbox (Vue, TS, etc.) and add the property, it does not work. However, if I fork the sandbox and add the property it does work. No idea why this would be the case.

**To Reproduce**
1. Go to https://codesandbox.io/p/sandbox/rg-start-vue-3-options-ap-forked-yrkx9z
2. Add `stretch="true"` to the `v-grid`
3. See the preview update and not work correctly.

Working using fork:

https://codesandbox.io/p/sandbox/rg-start-vue-3-options-ap-forked-6ywglt?workspaceId=5d410896-4a34-41e8-a3c2-c11144e83e18

**Comment by MattTheCuber** [comment]
Oh, is this a Pro feature?

**Comment by revolist** [comment]
Hi @MattTheCuber!

Thanks for bringing this up! It looks like you’re encountering some differences between the features available in our Pro version and the open-source version.

	1.  Pro Version: The stretch plugin  is designed to work across all columns and stretch columns by indexes.
	2. Open Source Version: In the open-source version, we made some adjustments, but it may not handle all stretching scenarios effectively. But it does have some limitations in the open-source version. Specifically, it may not always detect changes automatically, and you might need to manually update the columns for it to function correctly. Also it's stretches just the last column.

To help illustrate how stretching works in the core version, I created an example you can check out: [Stretch Example in Free Version](https://codesandbox.io/p/sandbox/rg-start-vue-3-options-ap-forked-9lmc72).

If you have any further questions or need additional assistance, feel free to reach out!
